src/test/java/com/lorepo_patchers/icplayer/client/PlayerControllerPatcher.java

```diff
@@ -10,6 +10,11 @@ public class PlayerControllerPatcher {
 	public static int getIFrameScroll (PlayerController a, PlayerController x) {
 		return 0;
 	}
+
+	@PatchMethod
+	public static int handleCurrentPageIdRequest (PlayerController a, PlayerController x) {
```

Comment:
Dlaczego ta metoda przyjmuje dwa argumenty, skoro `handleCurrentPageIdRequest` przyjmuje jeden ?

Reply:
jest to niezbędne do poprawnego wykonania testów

Reply:
dodana metoda "handleCurrentPageIdRequest" jest napisana w GWT, co powoduje problemy w testach javy, w tym celu dodano mock-a tej metody oraz celowo ustawiono typ zwracany